src/App.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react';
```

Comment:
I don't think this file is needed to fulfil the brief. Can it be removed?

Reply:
How so? The useState hook is required to create a state in the functional component.

Reply:
Oh! Sorry, I mean `src/App.js`. Often, when I'm making a comment relating to the file as a whole, I will leave the comment on the first line of the file.

So my previous statement means. I don't think `src/App.js` is needed to fulfil the brief. Can `src/App.js` be removed from this change set? (Please also remove any related files as well e.g `src/App.css`)